### DISCUSSION TITLE
Bind to IPv6

### DISCUSSION THREAD

**Discussion by mattoni** [remix-run/remix#2762] [Q&A] [answered]
I was unable to find how to force a dual stack binding (`::`) so that the remix server will also listen on ipv6. Is there a way to achieve this? Thanks!

**Comment by sergiodxa** [comment, marked as answer]
If you need to customize the server you can switch to setup Express yourself.

**Reply by mattoni** [reply]
Hmm, seems a bit much for just changing a bind address, but I understand. Is there any plan to make the bind address changeable via env variable, or enable ipv6? Definitely cheaper to use IPv6 where possible.

Thanks!

**Reply by gatesn** [reply]
I've found setting `HOST=::` seems to work.

```
Remix App Server started at http://localhost:8081 (http://:::8081)
```
